foundation/aissemble-foundation-core-python/src/aissemble_core_config/rdbms_connection_pool.py

```diff
@@ -0,0 +1,9 @@
+###
+# #%L
+# aiSSEMBLE::Foundation::Core Python
+# %%
+# Copyright (C) 2021 Booz Allen
+# %%
+# This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+# #L%
+###
```

Comment:
Do we need this blank file?

Reply:
Removed this file, was added in as part of a merge conflict.